src/runtime/threads/executors/thread_pool_executors.cpp

```diff
@@ -339,14 +339,14 @@ namespace hpx { namespace threads { namespace executors { namespace detail
 
             // FIXME: turn these values into performance counters
             std::int64_t executed_threads = 0, executed_thread_phases = 0;
-            std::uint64_t overall_times = 0, thread_times = 0;
+            std::uint64_t overall_times = 0, thread_times = 0, bg_work=0;
```

Comment:
Please follow the formatting we use everywhere else in this file (use spaces surrounding `=`)